app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CustomerItemsController < ApplicationController
+  def new
```

Comment:
We don't need this action since the form is being rendered from the Customers controller, show action.